assets/src/components/stacks/overview/StackConfiguration.tsx

```diff
@@ -1,25 +1,31 @@
-import { Button, Card, FormField, Input } from '@pluralsh/design-system'
+import { Button, Card, FormField, Input, Switch } from '@pluralsh/design-system'
+import LoadingIndicator from 'components/utils/LoadingIndicator'
 import { useState } from 'react'
 import { useOutletContext, useParams } from 'react-router-dom'
-import LoadingIndicator from 'components/utils/LoadingIndicator'
 import { useTheme } from 'styled-components'
 
 import { useUpdateStackMutation } from '../../../generated/graphql'
 import { GqlError } from '../../utils/Alert'
+import { OverlineH1 } from '../../utils/typography/Text'
 
 import { StackOutletContextT } from '../Stacks'
-import { OverlineH1 } from '../../utils/typography/Text'
 
 export default function StackConfiguration() {
   const theme = useTheme()
   const { stackId = '' } = useParams()
   const { stack, refetch } = useOutletContext() as StackOutletContextT
   const [image, setImage] = useState(stack.configuration.image)
   const [version, setVersion] = useState(stack.configuration.version)
+  const [parallelism, setParallelism] = useState(
```

Comment:
these fields should only be set if its a terraform stack, i think nick's mocks did a poor job of conveying that though

Reply:
ahh... that makes sense. I will hide them if it's not tf.